pkg/roachprod/vm/gce/utils.go

```diff
@@ -211,6 +193,31 @@ for service in apport.service atd.service; do
   systemctl mask $service
 done
 
+# Enable core dumps, do this last, something above resets /proc/sys/kernel/core_pattern
+# to just "core".
+cat <<EOF > /etc/security/limits.d/core_unlimited.conf
+* soft core unlimited
+* hard core unlimited
+root soft core unlimited
+root hard core unlimited
+EOF
+
+mkdir -p /mnt/data1/cores
+chmod a+w /mnt/data1/cores
+cat <<'EOF' > /bin/gzip_core.sh
+#!/bin/sh
+exec /bin/gzip -f - > /mnt/data1/cores/core.$1.$2.$3.$4.gz
```

Comment:
`pigz` might save a few mins if the core is very large. (May need to `apt-get install pigz`.)

Reply:
For a 300MB core its like 3s vs .3s so wow, that's like a 10x speedup but hardly seems worth it.